src/test/regress/sql/row_types.sql

```diff
@@ -66,4 +95,5 @@ EXECUTE rec(1);
 EXECUTE rec(1);
 EXECUTE rec(1);
 
+SET client_min_messages TO error;
```

Comment:
usually no point in doing a SET at the end of the file, since the session ends immediately after

Reply:
Stops `DROP CASCADE` listing everything